### ISSUE TITLE
kd loss not decrease

### ISSUE THREAD

**Issue by Rui-Zhou-2** [yzd-v/MGD#5]
Hi, have you tried to train from the scratch for student?

and plus...will your kd loss converge in the training process? 

When I try it for a segmentation task (resnet50-18), the kd loss not converge if I distil the (N,512,36,48) feature, thus not help the student....or only slightly change if I distil the first layer that is larger.

since my feature map is smaller, is this the direct reason?

would you kindly give me some idea on this?

With best regards

**Comment by yzd-v** [comment]
Maybe we can talk about this in a more effective way. (你的知乎设置了拒收陌生人消息

**Comment by Rui-Zhou-2** [comment]
是的。。你可以再回复我一遍吗。我刚刚打开了消息

**Comment by XJWang628** [comment]
Hello, 
I'm currently exploring the use of MGD for an instance segmentation task using Mask R-CNN. For this task, I attempted to train the student model from scratch. However, during the training process, I encountered an issue where the Knowledge Distillation (KD) loss did not converge, and I was unable to find a potential cause or solution. Could you provide any insights or recommendations on how to address this issue? Are there any specific hyperparameters or settings that could potentially affect the KD loss convergence when using MGD with Mask R-CNN? Any assistance would be greatly appreciated. Thank you for your time and support. @yzd-v @buble-pie

**Comment by XJWang628** [comment]
![image](https://github.com/yzd-v/MGD/assets/74861454/d6595d60-9a4c-4400-80b1-e46e2f43509f)

**Comment by yzd-v** [comment]
Maybe the generative block is not add to the optimizer. You can check [this](https://github.com/yzd-v/MGD/blob/master/det/mmdet/apis/train.py#L131).

**Comment by XJWang628** [comment]
Thank you for your reply, it’s very helpful to me
